src/Symfony/Component/Dotenv/Command/DotenvDumpCommand.php

```diff
@@ -43,7 +43,7 @@ public function __construct(string $projectDir, string $defaultEnv = null)
     /**
      * {@inheritdoc}
      */
-    protected function configure()
+    protected function configure(): void
```

Comment:
we decided to not add `void` return types to existing methods, because this also doesn't provide anything in practice for the codebase, this might break BC (not here because the class is final) and thus this creates needless merge conflicts.

Reply:
Return type removed